src/main/java/appeng/block/AEBaseTileBlock.java

```diff
@@ -241,9 +240,13 @@ public boolean recolorBlock( final World world, final BlockPos pos, final EnumFa
 	public int getComparatorInputOverride( IBlockState state, final World w, final BlockPos pos )
 	{
 		final TileEntity te = this.getTileEntity( w, pos );
-		if( te instanceof IInventory )
+		if( te instanceof AEBaseInvTile )
 		{
-			return Container.calcRedstoneFromInventory( (IInventory) te );
+			AEBaseInvTile invTile = ( AEBaseInvTile ) te;
+			if ( invTile.getInternalInventory() != null && invTile.getInternalInventory().getSlots() > 0)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)